lib/genre.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './item'
+
+class Genre < Item
+  attr_accessor :name
+
+  def initialize(name)
+    super()
+    @id = Random.rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item; end
```

Comment:
It would be better if you implemented the add-item method for the genre class. You can follow the instructions in the project requirements. I found this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md) very helpful when creating relationships in code